python/deltalake/table.py

```diff
@@ -77,6 +77,7 @@ def __init__(
         table_uri: str,
         version: Optional[int] = None,
         storage_options: Optional[Dict[str, str]] = None,
+        without_files: Optional[bool] = None
```

Comment:
Why not make it clear what the default is in the signature?

```suggestion
        without_files: bool = False
```